### DISCUSSION TITLE
Connections to Postgres directly from an edge function are secured with SSL

### DISCUSSION THREAD

**Discussion by laktek** [supabase/supabase#18845] [Changelog]
If you use Deno Postgres or other Postgres clients to connect to your database instance from a Supabase Edge Function, those connections are now secured with SSL. You don't need to add any extra configurations to your client setup.

More details:  https://supabase.com/docs/guides/functions/connect-to-postgres#ssl-connections

**Comment by akdeb** [comment]
@laktek My MCU ESP32 client requires a RootCA certificate to connect to the SSL Supabase Edge Function. How do you obtain that?
